app/views/enrollment_mailer/api_entreprise/review_application.text.erb

```diff
@@ -0,0 +1,13 @@
+Bonjour <%= "#{@user.first_name} #{@user.last_name}" %>,
+
+Votre demande d’habilitation n°<%= @enrollment.id %> a été traitée par notre service juridique, il nous manque cependant certaines informations pour rendre un avis.
+
+Votre demande requiert les modifications suivantes :
+
+• XXXXXX
```

Comment:
It's simpler to keep this zone empty I think. I'd like to be able to set the default cursor position at this place in the front end... 🤔

Reply:
> It's simpler to keep this zone empty I think. I'd like to be able to set the default cursor position at this place in the front end... 🤔

I just c/c the template from Dorine fyi. I can add a special string for the cursor. 

Maybe `<cursor>` ?

Reply:
perfect

Reply:
a59a933